plugins/autocertkit

```diff
@@ -464,27 +465,8 @@ def validate_xml_key(key):
     else:
         return False
 
-
-def to_xml(d, t):
-    s = '<?xml version="1.0"?>\n<%s' % t
-    for k, v in d.iteritems():
-        if not validate_xml_key(k):
-            continue
-        s += ' %s="%s"' % (k, xmlrpclib.escape(v))
-    s += '></%s>\n' % t
-    return s
-
-def list_to_xml(l, t):
-    s = '<?xml version="1.0"?><%ss>\n' % t
-    for d in l:
-        s += '<%s' % t
-        for k, v in d.iteritems():
-            if not validate_xml_key(k):
-                continue
-            s += ' %s="%s"' % (str(k).replace(' ','_'), xmlrpclib.escape(str(v)))
-        s += '></%s>\n' % t
-    s += '</%ss>' % t
-    return s    
+def json_dumps(data):
+    return json.dumps(data, indent=2)
```

Comment:
This does not really matter as we won't hand parse this json. But I would recommend  you to use slightly prettier output. i.e. json.dumps(manifest, indent=2, separators=(',', ': '))